README.md

```diff
@@ -630,3 +631,24 @@ One of three out comes can occur after the export retried has completed.
 
 > *NOTE: Changing Writable.Pipeline.ExecutionOrder will invalidate all currently stored data and result in it all being removed from the database on the next retry.* This is because the position of the export function can no longer be guaranteed and no way to ensure it is properly executed on the retry.
 
+### Secrets
+
+Application Services can retrieve their secrets from the underlying secret store using the GetSecrets() API in the SDK. When security is disabled, the secrets are retrieved from the Writable.InsecureSecrets section of the configuration file. An example of the configuration for insecure secrets is below.
+
+#### Insecure Secrets Configuration
+
+```toml
+ [Writable.InsecureSecrets]    
+      [Writable.InsecureSecrets.CloudSecrets]
+        Path = '/v1/secret/edgex/appservice/cloud'
+        [Writable.InsecureSecrets.CloudSecrets.Secrets]
+          username = 'cloud'
+          password = 'cloudPw'
+
+      [Writable.InsecureSecrets.DBSecrets]
+        Path = '/v1/secret/edgex/appservice/db'
```

Comment:
Same path change, but here is would be `mongo` or `redis`. This is just the way they namespace'd them in Vault. Also, if we put the db creds here we have to modify GetDatabaseCredentials() to pull from here when running non-secure mode, which I was thinking about and think is the correct thing to do.

Reply:
yea, i was going to ask about that, since it currently pulls from the Database section of the config file. I'll make the change to pull the Database credentials as part of this PR.